scripts/obtain/uninstall/dotnet-uninstall-pkgs.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
```

Comment:
@blackdwarf can you take a look? This will remove all dotnet installation in `/usr/local/share/dotnet`